### DISCUSSION TITLE
Mention of the `csquotes` package

### DISCUSSION THREAD

**Discussion by pablgonz** [learnlatex/learnlatex.github.io#115] [Ideas]
Some of the usual tools used when writing a text are:

- `\textbf`, `\textit`, `\emph`
- `(...)` `[...]`
- `"..."`, ` '...'`

The first two points are common to most languages, but the last one is quite variable. The first document with which I started to learn a little LaTeX was the classic one:

```latex
\documentclass{article}
\begin{document}
"Hola Mundo"
\end{document}
```
Just to note that _"quotation marks do not result in quotation marks"_. As this great project is being translated into several languages, I think it's worth making a special mention to the topic of "quotation marks", a quick glance at [Quotation mark](https://en.wikipedia.org/wiki/Quotation_mark) makes me think that it would be a good idea (thinking of those who are just starting out), to know how to get the quotation marks appropriate to their own language.

I write this here as an idea.

**Comment by josephwright** [comment]
Roughly where you put this?

**Reply by pablgonz** [reply]
`lesson-14`?

**Reply by pablgonz** [reply]
or in `more-03` if quotation marks can be considered as _special characters_

**Reply by josephwright** [reply]
Let's see what others think!

**Reply by PhelypeOleinik** [reply]
Maybe in the language-specific lessons, since it will change (at least a bit) from one language to another?
